contributing.md

```diff
@@ -36,20 +36,32 @@ one or two development branches are actively maintained.
 Running Tests
 -------------
 
-*Note:* This section needs better instructions.
-
 Run `tox` from within your checkout. This will run the tests across all
 supported systems and attempt to build the docs.
 
-To run the tests for Python 2.x only:
+For example, to run the test suite with your current Python runtime:
+
+    tox -e py
+
+If you wish to run multiple targets, you can do so by separating them with a
+comma:
+
+    tox -e py312,coverage
+
+If you've already run the coverage target and want to see the results in HTML,
+you can override `tox.ini` settings with the `commands` option:
+
+    tox -e coverage -x testenv:coverage.commands="coverage html"
+
+To build the docs:
 
-    $ tox py2-cover
+    tox -e docs
```

Comment:
```suggestion
.. code-block:: shell

    tox -e docs
```

Reply:
Isn't this docutils syntax, whereas this file is markdown?

Reply:
Corrected syntax. Thanks!